.eslintrc.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  'env': {
+    'browser': true,
+    'jquery': true
+  },
+  'extends': 'standard',
+  'parserOptions': {
+    'ecmaVersion': 5
+  },
+  'rules': {
+    // -----------------------------------------------------------------------------------------------------------------
+    // Overrides 'standard' rules to suit the present format
+    // -----------------------------------------------------------------------------------------------------------------
+    'brace-style': ['error', 'stroustrup', { 'allowSingleLine': true }],
+    'indent': ['error', 2, { 'outerIIFEBody': 0 }],
```

Comment:
I think the indent should be set to 4 characters instead of 2, just because var/let groups look ugly otherwise:
```javascript
var a,
    b,
    c,
    d;
```

would become 

```javascript
var a,
  b,
  c,
  d;
```

as seen in https://github.com/fomantic/Fomantic-UI/pull/555/commits/aea64d9b546961366b71ab8f60c8599d566f5da4#diff-77caea316b6e2c075a2c5d36f572b036  (weekOfYear function)

Reply:
Instead of debating `2`-VS-`4` just for `var` alignment, I suggest we should 

1. Keep 2 chars indentation as-is 
2. Rewrite single var declarations into separate `var` declrations by enabling `'one-var': 'never'`, in subsequent PR

I propose these because I believe `one-var: never` is better choice for maintainability from other JavaScript/TypeScript OSS experiences..


# Reason 1: Clearer diff when adding/removing variables

## one-var 👎 

```diff
 var a,
-    b = true,
-    c = b.bar();
+    c = b.bar(),
+    d;
```

## no one-var 👍 

```diff
  var a;
- var b = true;
  var c = b.bar();
+ var d;
```

# Reason 2: Future-proof (`let` and `const`)

We will eventually use `let` and `const` when we adopt ES6 #319 or TypeScript #456 or similar.

Though [ESLint is very configurable](https://eslint.org/docs/rules/indent#enforce-consistent-indentation-indent), but I don't think it is good idea to chnge indentations like this (`4`for `var`/`let`, `6` chars for `const`)

```js
let a,
    b = true;
const c = $('.foo')
      d = c.bar();
d.buz();
```

or this (`6` for all)

```js
let   a,
      b = true;
const c = $('.foo')
      d = c.bar();
d.buz();
```

The below `one-var: nver` style makes each variable declarations more obvious.

```js
let a;
let b = true; 
const c = $('.foo');
const d = c.bar();
```

Reply:
We don't need to debate on 2/4, it was just my personal feeling, so i would still be fine with it, if everybody else also agrees.

But I disagree with using `no one-var`. Having lots of vars is bad style and IMHO not very much maintainable or readable and would also mean every class var definition would need to be changed aswell, so
```javascript
var a,
    b= function(){
    },
    c= function()= {
    },
    d= function() = {
    };
```
would become
```javascript
var a;
var b= function(){
  };
var c= function()= {
  };
var d= function() = {
 };
```
Even removing / commenting vars does not make any difference

So i dont think we should use `one-var: never`

Reply:
Anyhow, as you said, a possible change to the handling of "vars/lets/const" would be part of an later PR, i approve this now 🙂